### PR TITLE
Stop and disable iscsi-starter.service

### PR DESCRIPTION
**Pull request by knoha-rh**
When an EDPM node is rebooted  which runs a VM with iscsi-backed volume, iscsi-starter.service is started on the node then iscsid.service is started via running systemctl start iscsi.service in iscsi-starter.service.

To prevent the issue, we need to stop/disable iscsi-starter.service.

JIRA: [OSPRH-12372](https://issues.redhat.com/browse/OSPRH-12372)

### CODE REVIEW

**Comment by rabi** [comment]
@knoha-rh Maybe you can request for you to be added to the organization in #openstack-k8s-operators.

**Comment by knoha-rh** [comment]
@rabi and @frenzyfriday 

I'm sorry for interrupting you. It will be great if you review this patch and give your approval on it.
Would you please review this patch?

**Review by rabi — COMMENTED** [review]
/lgtm

File: roles/edpm_iscsid/tasks/install.yml [hunk 1]
```diff
@@ -55,3 +55,13 @@
         - ansible_facts.services["iscsi.service"] is defined
         - ansible_facts.services["iscsi.service"]["status"] != "not-found"
         - ansible_facts.services["iscsi.service"]["status"] == "enabled"
+
+    - name: Stop and disable iscsi-starter.service
```
[the remark this answers is not in the document]
Comment (knoha-rh) [reply roles/edpm_iscsid/tasks/install.yml:59]: Oh... I need to update it. I'll do it.

**Pull request closed without merging**